arbiter-core/src/middleware.rs

```diff
@@ -167,7 +173,7 @@ impl MiddlewareError for RevmMiddlewareError {
     }
 
     fn as_inner(&self) -> Option<&Self::Inner> {
-        Some(self)
+        None
```

Comment:
Can you explain this to me? The middleware onion has been a little black boxed for to only handle it on one layer at a time.